application/phoenix/README.md

```diff
@@ -0,0 +1,35 @@
+This Phoenix example checks if the environment variables `DEMO_USERNAME` and `DEMO_PASSWORD` have been set. If that's the case, you'll receive a personalized greeting message on http://localhost:4000, otherwise you will not be recognized.
```

Comment:
```suggestion
This Phoenix example checks if the environment variables `DEMO_USERNAME` and `DEMO_PASSWORD` have been set. If that's the case, you'll receive a personalized greeting message on http://localhost:4000, otherwise you will get a "Hello stranger" message.
```

I think a more specific sentence would be better here.

Reply:
fixed